lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
```

Comment:
Are we using it in other places?  I'm fine with using it since you will be able to fix anything needed.